doc/manual/manual.tex

```diff
@@ -9567,14 +9567,14 @@ \subsubsection{Thin shell gravity benchmark}
 -164391.6151 &
 -164392.2246 &
 -164390.9963 \\
-&+1 & $3^3$ &  
+& 1 & $3^3$ &  
```

Comment:
Is this n_q? Would it be helpful to add a header to this table?

Reply:
oups, for some reason there I left out 2 '+' signs which serve no purpose. And also the values in the column below are 0.